doc/doxygen/src/Examples.dox

```diff
@@ -64,7 +64,12 @@
  * that some example @ref asset_management_system has a plugin,
  * installed on @ref plugin_path_var).
  *
- * We will be providing an example manager implementation soon!
+ * Check out <a href="https://github.com/OpenAssetIO/Template-OpenAssetIO-Manager-Python"
```

Comment:
[fix] I'd be happy just keeping the removal and no need to link the Template or Ayon.

If you want to keep these links, can we change `OpenAssetIO Manager Python` to `Python Manager Template`?

Reply:
I'm personally partial to adding additional cross-references in documentation so people can more easily find things. Let me know if you disagree though!